doc/api/dns.md

```diff
@@ -895,6 +914,18 @@ Here is an example of the result object:
     minttl: 60 } ]
 ```
 
+## `dnsPromises.resolveCaa(hostname)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `hostname` {string}
+
+Uses the DNS protocol to resolve `CAA` records for the `hostname`. The
+`addresses` argument passed to the `callback` function
```

Comment:
This description needs to be updated since it's dealing with a Promise, not a callback.

Reply:
Documentation has been updated.